src/Modules/common/CodeEditor/CodeEditor.tsx

```diff
@@ -14,6 +14,7 @@ interface ICodeEditorProps {
     code: string;
     codeEditorLang: languageObjectType['lang'];
     handleCodeChange: (value: string) => void;
+    isReportPage: boolean;
```

Comment:
instead of using `isReportPage` u can use something generic to the CodeEditor component like `hideLanguageSelection`